code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -23,6 +23,7 @@
 	var/leaping = 0
 	ventcrawler = 2
 	var/list/alien_organs = list()
+	emote_types = list("alien", "carbon", "living", "mob")

```

Comment:
The magic strings everywhere oh god
Why do emotes not define what mobs get them? Why do mobs have to define general categories?